webpack.config.js

```diff
@@ -1,7 +1,22 @@
+const path = require('path')
+
+const webpack = require('webpack')
+
 module.exports = {
-  externals: {
-    fs: 'fs',
-    https: 'https',
-    url: 'url'
-  }
+  entry: './src/index.js',
+  output: {
+    path: path.resolve(__dirname, 'dist'),
+    filename: 'solid-auth-tls.min.js',
+    library: 'SolidAuthTLS',
+    libraryTarget: 'umd'
+  },
+  module: {
+    rules: [
+      { test: /\.js$/, exclude: /node_modules/, loader: "babel-loader" }
+    ]
+  },
+  plugins: [
+    new webpack.DefinePlugin({ 'global.IS_BROWSER': true }),
```

Comment:
Now clients just have to add this single line to their webpack config rather than the list of externals above. This is the best strategy I've seen thus far for keeping implementation details of isomorphic libraries out of client code.